src/data-access/repositories/mongo/mongo-enums.ts

```diff
@@ -0,0 +1,6 @@
+export enum MongoChangeEvent {
+  INSERT = 'insert',
```

Comment:
Are we okay with the value being lowercased?

Reply:
Not sure if we have a choice. This is the value that MongoDB emits to indicate what event has happend.
I can try and see if I can make it uppercase. That or maybe an enum is the wrong choice here?

Reply:
The values must be lowercased to match the values in Mongo. So unless we want to do some case-gymnastics, this seems to be the simplest way.
That or we should just make each a constant?

Reply:
Personally, I am fine with having the enum values lowercased.